static/js/info_overlay.js

```diff
@@ -162,7 +292,28 @@ export function set_up_toggler() {
     $markdown_help.find(".rendered_markdown").each(function () {
         rendered_markdown.update_elements($(this));
     });
+
+    for (const row of search_operator_rows) {
+        // We already translated the html note and keyword operator row
+        // in `search_operator_rows`.
+        if (!(row.note_html || row.operator_key === "keyword")) {
+            const translated_key = $t_html({defaultMessage: "{key}"}, {key: row.operator_key});
+            const translated_value = $t_html(
+                {defaultMessage: "{value}"},
+                {value: row.operator_value},
+            );
+            const translated_effect = $t_html({defaultMessage: "{effect}"}, {effect: row.effect});
```

Comment:
Translation can’t be parameterized like this. All translators will see here is `{key}` and `{effect}`.

Reply:
So, I guess [this is wrong](https://github.com/zulip/zulip/blob/0b4338b99823590f4c65a18c878a90397707879e/static/js/settings_invites.js#L207) too?

Reply:
No. `$t_html({defaultMessage: "Revoke invitation to {email}"}, {email})` is an ordinary correct use of translation. The translator sees the string “Revoke invitation to {email}” and translates it to say “Einladung an {email} widerrufen”.

Your `$t_html({defaultMessage: "{effect}"}, {effect: row.effect})` does not work. The translator sees nothing but the string “{effect}”. There is nothing to translate there. The translator has no opportunity to see the strings like “Narrow to messages on stream”, because they’re not marked for translation.